README.md

```diff
@@ -6,16 +6,24 @@ Apache SkyWalking Kubernetes
 To install and configure skywalking in a Kubernetes cluster, follow these instructions.
 
 ## Documentation
-#### Deploy SkyWalking and Elasticsearch (default)
+#### Deploy SkyWalking and Elasticsearch 7 (default)
```

Comment:
> > 1. Why is the default es version 7 since the default docker image is 6? It might confuse the user.
> 
> skywalking-es6 -> skywalking oap image 6.6.0-es6 and es 6
> skywalking-es7 -> skywalking oap image 6.6.0-es7 and es 7
> 
I mean in README.md, you hint user that we support ES 7 as default dependency, but our `apache/skywalking-oap-server:6.6.0` image points to ES 6.

Reply:
got it.